Reneiski_Homework1/ReneiskiSelenium11/PageObjects/Pages/WebTablePage.cs

```diff
@@ -0,0 +1,38 @@
+﻿using OpenQA.Selenium;
+using ReneiskiSelenium11.Common.WebElements;
+using ReneiskiSelenium11.Data;
+using ReneiskiSelenium11.Data.Constants;
+
+namespace ReneiskiSelenium11.PageObjects.Pages
+{
+    public class WebTablePage : BasePage
+    {
+        private MyWebElement WebTablePagePath = new MyWebElement(By.XPath("//*[contains(text(), 'Web Tables')]"));
+        private MyWebElement TextRslt = new MyWebElement(By.XPath("//*[@class='rt-tbody']/div[4]"));
+        private MyWebElement AddButton = new MyWebElement(By.XPath("//*[@id='addNewRecordButton']"));
+        private MyWebElement SubmitButton = new MyWebElement(By.XPath("//*[@id='submit']"));
+        private MyWebElement EditButton = new MyWebElement(By.XPath("//*[@id='edit-record-4']"));
+        private MyWebElement DeleteButton = new MyWebElement(By.XPath("//*[@id='delete-record-4']"));
+
+        public string GetCellText(string value)
+        {
+            var xPath = string.Format(ConstantsWebTablePage.XPathToTableCell, value);
```

Comment:
better name it more specific, ex. cellXPath, cellElement, cellText

Reply:
Fixed